src/_playground/App.scss

```diff
@@ -1,3 +1,4 @@
+$mediaMaxWidth: 700px;
```

Comment:
I think this value should be altered a bit.  The SAP Concur HIG uses` 1280px` as the breakpoint for collapsing the left navigation.  I think if you go all the way down to `700px`, the content area is going to get really compressed.  Besides, we also have a story coming down the pipeline to add a table of contents which will likely end up as a right column (once again similar to the SAP Concur HIG).

Reply:
Okay I will change it. I believe angular had a max of 680px or something like that and I was modeling them.